nuxeo-importer-stream/src/main/java/org/nuxeo/importer/stream/consumer/BlobMessageConsumer.java

```diff
@@ -49,10 +52,13 @@
     public BlobMessageConsumer(String consumerId, String blobProviderName, BlobInfoWriter blobInfoWriter) {
         super(consumerId);
         this.blobProviderName = blobProviderName;
-        this.blobProvider = Framework.getService(BlobManager.class).getBlobProvider(blobProviderName);
-        if (blobProvider == null) {
-            throw new IllegalArgumentException("Invalid blob provider: " + blobProviderName);
+        if (!StringUtils.isBlank(blobProviderName)) {
+            this.blobProvider = Framework.getService(BlobManager.class).getBlobProvider(blobProviderName);
```

Comment:
Remove `this.`, it's a bit confusing as we have a usage without it the line below.